packages/local-environment/src/nomadenv.ts

```diff
@@ -141,6 +145,23 @@ export class NomadEnv {
     return false;
   }
 
+  get forkUrl(): string | undefined {
+    if (process.env.ALCHEMY_API_KEY)
+      return `https://eth-mainnet.g.alchemy.com/v2/${process.env.ALCHEMY_API_KEY}`;
+  }
+
+  get wETHAddress(): string {
+    return `${process.env.WETH_ADDRESS}`;
+  }
```

Comment:
Also this, weth not guaranteed to have the same address in all nets.

Reply:
Removed / replaced with weth

Reply:
*from process.env on creation

Reply:
Removed